README.md

```diff
@@ -5,26 +5,41 @@ ORD Water Distribution Network Model (AKA "EPANET")
 
-EPANET performs extended period simulation of hydraulics and water quality within potable water distribution networks. EPANET source code is written in the C Programming Language and released in the Public Domain.
+EPANET was developed as a tool for understanding the movement and fate of drinking water constituents within distribution systems, and can be used for many different kinds of applications in distribution systems analysis. Today, engineers and consultants use EPANET to design and size new water infrastructure, retrofit existing aging infrastructure, optimize operations of tanks and pumps, reduce energy usage, investigate water quality problems, and prepare for emergencies. EPANET can also be used to model contamination threats and evaluate resilience to security threats or natural disasters.
+
+This is the official EPANET source code repository maintained by the United States Environmental Protection Agency (EPA). EPANET source code is written in the C Programming Language and released in the Public Domain.
 
-Lewis Rossman created EPANET in the early 1990s and actively developed and maintained the project until 2008. Over the intervening years he received support from too many members of the WDS modeling community to name individually. 
- 
 Intended Audience
 -----------------
 The intended audience for this repository can be divided roughly into two groups:
  
  - Programmers interested in water distribution system modeling
  - Engineers working on water distribution system modeling interested in programming
  
-Git Hub and software repositories are geared towards people with software development experience. If you've ever hacked EPANET or built a Matlab script for running hydraulic simulations, you may find the contents of this repository helpful. We encourage you to fork EPANET and develop a great new feature to share with the community. 
+EPANET has an active open source software community including consulting engineers, students, researchers, software companies, professional organizations, other interested members of the public, and EPA partners. An Open Source EPANET Initiative at http://community.wateranalytics.org has released source code for an EPANET engine version 2.1 that included community sourced bug fixes and they are working on additional releases with new feature requests. Others are encouraged to fork EPANET and develop a great new feature to share with the community. 
+
+License
+-------------
+EPANET is released under the Creative Commons CC0 public domain license. See the LICENSE.txt file.
+
+Contributing to EPANET
+-------------
+Information on how to contribute to the development of EPANET can be found in the CONTRIBUTING.md file.
```

Comment:
is this file part of a PR? i cannot find it

Reply:
I think this PR wasn't for OWA

Reply:
right, i understand - the readme in this PR refers to `CONTRIBUTING.md`, but i cannot find that file anywhere here. I'm commenting on the USEPA repository as an interested observer 😄

Reply:
I was going to put in a new file into the repo called CONTRIBUTING.md, but I haven't done so yet. I plan to push another pull request later today with this file.